emData/download.sh

```diff
@@ -304,7 +228,7 @@ do
   echo ${module}
   memprint_location="MemPrintsSplit"
   memprint_location_reducedcm="MemPrintsReducedCM"
-  memprint_location_reducedcm2="MemPrintsReducedCM2"
+  ##memprint_location_reducedcm2="MemPrintsReducedCM2"
```

Comment:
@aryd I would prefer simply removing this line, since this configuration has been removed:
```suggestion
```

Reply:
Removed.